tests/functional/paging_test.php

```diff
@@ -35,7 +35,7 @@ public function test_pagination()
 		$this->assertNotContains('post no4', $crawler->text());
 		$this->assertContains('post no16', $crawler->text());
 
-		$prev_link = $crawler->filter('.pagination > ul > li.previous > a')->attr('href');
+		$prev_link = $crawler->filter('.pagination > ul > li.prev > a')->attr('href');
```

Comment:
sorry just common practice naming convention to use nxt and prev. can revert if need be